src/Windowing/Silk.NET.Windowing.Glfw/GlfwWindow.cs

```diff
@@ -246,7 +246,13 @@ protected override WindowBorder CoreWindowBorder
 
         protected override bool IsClosingSettable
         {
-            set => _glfw.SetWindowShouldClose(_glfwWindow, value);
+            set
+            {
+                if (_glfwWindow != null)
+                {
+                    _glfw.SetWindowShouldClose(_glfwWindow, value);
+                }
+            }
         }
 
         protected override Vector2D<int> SizeSettable
```

Comment:
We should probably guard similar calls too, but we can save that for another PR.